dists/flags/ubuntu.flags

```diff
@@ -11,6 +11,7 @@ notify-reboot-required complain
 package-system-locked attach_disconnected,complain
 packagekitd complain
 release-upgrade-motd complain
+setpriv complain
```

Comment:
I wonder how Arch behaves without this.

Reply:
I have never seen it on arch. Anyway, as it is not ubuntu/debian specific, you should move it in `dists/flags/main.flags`